web/styles/popovers.css

```diff
@@ -548,7 +548,8 @@ ul {
 
     #user-profile-streams-tab {
         .stream_list_info {
-            margin-bottom: 8px;
+            margin: 0;
+            background-color: transparent;
```

Comment:
We can avoid setting `background-color` here if we use `ui_report.message` and pass `text-success` class as a short term fix. A long-term fix can be to update `ui_report.success` to accept a class name as parameter with default being `alert-success`.